include/taskolib/Sequence.h

```diff
@@ -239,6 +256,20 @@ class Sequence
      */
     ConstIterator erase(ConstIterator first, ConstIterator last);
 
+    /**
+     * The sequence contains a common setup script that is shared by all of its steps. It
+     * is called automatically before the execution of each step's script, just after
+     * executing the lua_step_setup function from the context. This step setup script can
+     * be set with set_step_setup_script(). It is typically used like a small library for
+     * defining common functions or constants.
+     *
+     * However it is only relevant of the following Step types: ACTION, IF, ELSEIF, and WHILE.
+     *
+     * \param step_setup Step setup script executed before the execution of each Step's
+     *                   script.
+     */
+    void set_step_setup_script(std::string step_setup);
```

Comment:
I still object to using bare `std::string`s as parameters without a good reason for it. If we do that everywhere, it is wasteful.

Reply:
This I moved the data to `Context::step_setup` there is no need for this member function anymore.